react/src/components/molecules/ArrowNav/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {ArrowButton} from '../../../index';
+import './style.css';
+
+const ArrowNav = (props) => {
+  const sectionClasses = classNames({
+    'ma__arrow-nav': true,
+    [`ma__arrow-nav--${props.direction}`] : props.direction
+  });
+  const Element = props.href ? 'a' : 'button';
```

Comment:
This should be removed as it's not being used.

Reply:
Turns out this is needed for the update requested.